### PR TITLE
DBZ-2885 Fix test

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-2885

### CODE REVIEW

**Comment by jpechane** [comment]
@gunnarmorling Remaining test failure is fixed i https://github.com/debezium/debezium/pull/2182/commits/eebd16147dd1906913b3918b9f6050700e072ef6

File: debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java [hunk 1]
```diff
@@ -760,6 +760,14 @@ public OffsetStorageReader offsetStorageReader() {
                         connectorCallback.ifPresent(DebeziumEngine.ConnectorCallback::taskStarted);
                     }
                     catch (Throwable t) {
+                        // Clean-up allocated resources
+                        try {
+                            LOGGER.debug("Stopping the task");
+                            task.stop();
+                        }
+                        catch (Throwable tstop) {
+                            LOGGER.info("Error while trying to stop the task");
```
Comment (gunnarmorling) [inline debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java:769]:
```suggestion
                            LOGGER.error("Error while trying to stop the task");
```

Same for the other one perhaps?

Comment (jpechane) [reply debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java:769]: I intentionally set the `info` level as this is error that is just ignored

**Comment by gunnarmorling** [comment]
Applied, thx!

**Pull request merged**